### PR TITLE
[WIP] [ADD] Service Teams

### PR DESCRIPTION
**Pull request by dreispt**
Implements #5 and #17

### CODE REVIEW

**Review by robertrottermann — COMMENTED** [review]
I do not like the idea of assigning states to a team.
I discussed that with my people and I could find no good reason.
I think stages should be assigned to work-items (sub parts of a workorder)

File: fieldservice/models/fsm_stage.py [hunk 1]
```diff
@@ -12,20 +12,51 @@
 ]
 
 
+
+STATES = [
+    ('new', 'New'),
+    ('confirmed', 'Confirmed'),
+    ('scheduled', 'Scheduled'),
+    ('assigned', 'Assigned'),
```
Comment (max3903) [inline fieldservice/models/fsm_stage.py:20]: `('planned', 'Planned'),`

File: fieldservice/views/menu.xml [hunk 2]
```diff
@@ -87,11 +87,18 @@
               sequence="20"
               groups="base.group_no_one"/>
 
+    <menuitem id="menu_fsm_team"
+              name="Service Teams"
+              action="action_fsm_team"
+              parent="config"
+              sequence="30"
+              groups="base.group_no_one"/>
```
Comment (max3903) [inline fieldservice/views/menu.xml:95]: `groups="group_fsm_team"`

File: fieldservice/views/fsm_team.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <field name="res_model">fsm.team</field>
+        <field name="view_id" ref="fsm_team_tree_view"/>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="help" type="html">
+            <p class="oe_view_nocontent_create">
+                Create a Field Service Team.
+            </p>
+            <p>
+                Each Team addresses adiffernt type of work, and ca have their specific workflows.
```
Comment (max3903) [inline fieldservice/views/fsm_team.xml:42]: `Each team addresses different type of work, and can have their specific workflow.`

File: fieldservice/models/fsm_order.py [hunk 4]
```diff
@@ -12,13 +12,15 @@ class FSMOrder(models.Model):
     _inherit = ['mail.thread', 'mail.activity.mixin']
 
     def _default_stage_id(self):
-        return self.env.ref('fieldservice.fsm_stage_new')
+        return self.team_id.stage_ids[0]
 
+    team_id = fields.Many2one('fsm.team', string='Service Team')
```
Comment (max3903) [inline fieldservice/models/fsm_order.py:17]: @dreispt We want a default team and the team field on the order views should be seen only by the group_fsm_team group.

**Review by max3903 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**